Jellyfin.Api/BaseJellyfinApiController.cs

```diff
@@ -9,5 +10,23 @@ namespace Jellyfin.Api
     [Route("[controller]")]
     public class BaseJellyfinApiController : ControllerBase
     {
+        /// <summary>
+        /// Splits a string at a seperating character into an array of substrings.
+        /// </summary>
+        /// <param name="value">The string to split.</param>
+        /// <param name="separator">The char that seperates the substrings.</param>
+        /// <param name="removeEmpty">Option to remove empty substrings from the array.</param>
+        /// <returns>An array of the substrings.</returns>
+        internal static string[] Split(string value, char separator, bool removeEmpty)
```

Comment:
I migrated this method from `MediaBrowser.Api.ApiEntryPoint` but am uncertain if this is the right place to migrate it to.
As this method is used in multiple classes, I decided against a private method in `SearchController.cs`

Reply:
I would move it to a Helpers file, like in #2967

Reply:
I moved the method to the same class created in #2967 as creating a new class seemed to be unnecessary